index.html

```diff
@@ -1350,7 +1350,6 @@ <h4>Layout of <code>&lt;mtext&gt;</code></h4>
                 has an entry in the <a>MathTopAccentAttachment</a> table
                 then the specified value is used as the top accent attachment of
                 the <code>&lt;mtext&gt;</code> element.
-	      </li>
```

Comment:
this is removing the one just added???, might be simpler to have one PR with all diffs rather than a pr per line change